### ISSUE TITLE
Regression: adding-field-with-default rule not triggered for Postgres <=11 after v2.0

### ISSUE THREAD

**Issue by ermakov-oleg** [sbdchd/squawk#636]
After upgrading from squawk 1.x to 2.x, the adding-field-with-default rule no longer triggers for Postgres versions <=11.

**Expected behavior (v1.6.1):**

```bash
❯ squawk --version
squawk 1.6.1

❯ squawk --pg-version=10 <<EOF
ALTER TABLE my_table ADD COLUMN foo BOOLEAN NOT NULL DEFAULT FALSE;
EOF
stdin:1:0: warning: adding-field-with-default

   1 | ALTER TABLE my_table ADD COLUMN foo BOOLEAN NOT NULL DEFAULT FALSE;

  note: Adding a field with a VOLATILE DEFAULT requires a table rewrite with an ACCESS EXCLUSIVE lock. In Postgres versions 11+, non-VOLATILE DEFAULTs can be added without a rewrite.
  help: Add the field as nullable, then set a default, backfill, and remove nullability.

find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 1 issue in 1 file (checked 1 source file)
```

**Actual behavior (v2.24.0):**

```bash
❯ squawk --version
squawk 2.24.0

❯ squawk --pg-version=10 <<EOF
ALTER TABLE my_table ADD COLUMN foo BOOLEAN NOT NULL DEFAULT FALSE;
EOF

Found 0 issues in 1 file 🎉
```
